### ISSUE TITLE
Use ZMQ to monitor events from blockchain

### ISSUE THREAD

**Issue by peter-conalgo** [spesmilo/electrum-server#160]
Sure would be nice to use ZMQ to receive blocks and new transactions as they happen.

<https://github.com/bitcoin/bitcoin/blob/master/doc/zmq.md>

I'm not volunteering to do it, but it looks pretty easy and should reduce CPU load on server system.

**Comment by ecdsa** [comment]
please volunteer

**Comment by kamk** [comment]
Yes, I was thinking about this too. There's even [example](https://github.com/bitcoin/bitcoin/blob/master/contrib/zmq/zmq_sub.py) written in Python.
